frontend/src/components/HomePage/Filters.jsx

```diff
@@ -66,14 +60,24 @@ const Filters = ({ campaignList, setSearchTermsObj, setCampaignFilter }) => {
         }))}
         updateFilter={setCampaignFilter}
       />
+      {isFetching && (
+        <img
+          className="loading__icon"
+          src={loadingSvg}
+          alt="Loading indicator"
+        />
```

Comment:
```suggestion
        <div
          style={{
            display: 'flex',
            alignItems: 'end',
            paddingBottom: '.75rem',
          }}
        >
          <img
            className="loading__icon"
            src={loadingSvg}
            alt="Loading indicator"
          />
        </div>
```

I feel like this alignment makes more sense. Can also go in CSS.

<img width="594" alt="Screenshot 2024-09-04 at 5 17 51 PM" src="https://github.com/user-attachments/assets/be58eebb-4c9f-4e21-bff3-99f4737db6e7">